### ISSUE TITLE
RChain Community Validator Pools

### ISSUE THREAD

**Issue by jimscarver** [rchain/bounties#897]
With RChain proof-of-stake, validators are like miners in proof-of-work. They bet on transactions and are rewarded for good behaviour. It is planned that the coop will run validator nodes in addition to outside enterprizes.

There are members who would like to employ their REV on validators but do not want to operate nodes themselves. There are members who want to provide computing resources to a validator network but do not want to manage the network and security themselves.

There is the potential opportunity to have community managed validator pools supported by the coop and staked by the members.

There is a concern about conflict of interest in the coop competing against outside investment pools. At the same time it is healthy to further decentralize the network with more staking pools and validators. Engaging members in community managed validation educates members and can help fund community rchain usage.

Limiting the percentage of coop run validators might ease concerns while providing for self sufficiency in rchain usage for coop operations itself and greater decentralization rather than centralizing validation in the coop.

Management teams for pools could self identify, commit staking and become vetted by coop management in having the ability and processes to manage validators and distribute earnings according to established practices TBD. Coop support for the initial startup or expansion of the network might be obtained through the bounty system or other means.

If you are interested in leading or guiding RChain Community Validator Pools assign yourself to this issue. Together we might make them a reality.

**Comment by Valentine-Mario** [comment]
Is there a minimum about of resources someone could invest eg 500 GB?

**Comment by Ojimadu** [comment]
Is there any link to how pooling would work on RChain? Am yet to see any. So far I don't think there's a contract that can delegate staking like the way Tezos or Ark does theirs. 
The other option I see is a "2 of 3" multi-sig contract where the owner of the REV would have two keys and pool manager has one key 🗝. That way the pool manager can't sanction transactions without the signature of the owner of the REV but the owner of the REV can sanction transactions without needing the pool manager.

**Comment by Ojimadu** [comment]
@Valentine for high performant shards I saw on the validator sheet around 4tb RAM, 200TB disk space, 25GB/s ethernet plus some other requirements.

**Comment by David405** [comment]
@Ojimadu I don't think there is a hardware specification yet for validators ( @Valentine-Mario a 2GB ram with a decent processor should allow you run nodes with ease), I think the specs you made mention of is specifically related to the testnet in Germany and it still is a speculation. see [here](https://github.com/rchain/legaldocs/blob/master/validator-sale/Node%20Validation%20Overview%20DRAFT%20v1.docx)

**Comment by Ojimadu** [comment]
Yes, but I still expect similar hardware to deployed in high performant shards in the main net.

**Comment by Ojimadu** [comment]
_Coming from the RAM meeting yesterday, I said I would write down my statement._

I am still not sure how pool validation would work on the network other than requiring that participants send their REVs to a "trusted" validator to validate on their behalf. 
On Tezos and Ark blockchains participants are able to delegate their tokens to a particular pool operator without requiring that the tokens leave their wallets; that way, the participants don't bear the counterparty risk in the case where the private keys of the pool operator is compromised or in the case of a malicious pool operator.

**Comment by jimscarver** [comment]
I am glad to see interest in this issue. I see this as an experiment in cooperation and self-organization. If we are to succeed we must be organized and self organized grouyps tend to become dysfunctional as they grow.

At the member meeting I was tasked with helping to define the value proposition and risks. Video: https://youtu.be/9N752-0hBjk?t=30m38s Notes:

- Community validators pool @jim 
    - Allan/David: The issue at hand is that potential validators might not know “what they’re getting into”. Most don’t know the risks involved. 
    - what are the risks involved in validating? 
    - are there potential conflicts of interest in a community pool (probably not, but someone should address this and perhaps write down the analysis) 
    - Jim will create a clear statement of benefits and risks of a Community Validator Pool. 
    - Ojimadu will participate in the [#897](https://github.com/rchain/bounties/issues/897) discussion and make sure the answers to his technical questions are public in #897.

I created a [validator working group folder](https://drive.google.com/open?id=1crxkXQG1x6KLZKXWA1R_f0AeyyDGT-_U), a [working group document](https://docs.google.com/document/d/1qpodOFoA7AcZzoXTlc_ls0xWKIc39xIqVD1nvI7l62Q/edit) and [risk and rewards document](https://docs.google.com/document/d/1mQ8s2Ief9ob7viGBmkCWAZ63yaHP5KgX1KVZyKa7MXs/edit#)

Please participate and help lead this effort. If those who self assigned this issue take on some role We might succeed. We might get some answers to our questions in @kennyrowe office hours 7pm NY time in Description:https://zoom.us/j/7911989023 We might also devote some time to this issue in today's work studies at 2pm and 4pm NY time in https://zoom.us/j/6853551826

Who else on this team will be attending RCon3? I would be great have our ducks in a row by then and take part in the testnet genesis cooperatively.

**Comment by TrenchFloat** [comment]
I found the [RChain Validatory FAQ](https://rchain.atlassian.net/wiki/spaces/CORE/pages/490373121/RChain+validator+FAQ#RChainvalidatorFAQ-Stake), which seems like a better place to compile answer to questions than a whole new risk/reward google doc. However, most of its questions are just answered with a link to the [Validation Document](https://github.com/rchain/legaldocs/blob/master/validator-sale/Validation%20in%20the%20RChain%20Network.pdf) which is scary to read. I'll try to get in contact with @KellyAtPyrofex to see if we can edit the FAQ, make it a standalone doc, and keep from reinventing the wheel.

I might make this a separate issue. Let's see how it goes.
